nengo/spa/actions.py

```diff
@@ -154,7 +154,11 @@ class Actions(object):
     def __init__(self, *args, **kwargs):
         self.actions = None
         self.args = args
-        self.kwargs = kwargs
+        self.kwargs = OrderedDict(kwargs)
```

Comment:
What's the reason for the `OrderedDict`? It's perhaps a little misleading because `**kwargs` is not guaranteed to maintain order, so I'm not sure how you expect to use the ordering.